src/index.js

```diff
@@ -6,16 +6,21 @@ import { renderPreview, renderLoading, renderError, renderDisambiguation, render
 import { getWikipediaAttrFromUrl, isTouch, getDir, isOnline } from './utils'
 
 const invokeCallback = ( events, name, params ) => {
-	const callback = events && events[ name ]
-	if ( callback instanceof Function ) {
-		try {
-			callback.apply( null, params )
-		} catch ( e ) {
-			// eslint-disable-next-line no-console
-			console.log( 'Error invoking Wikipedia Preview custom callback', e )
+		const callback = events && events[ name ]
```

Comment:
Not sure why the indentation changed here.

Reply:
It's the linter autocorrecting because of https://eslint.org/docs/rules/one-var. Actually moving `version` to util should restore the original indentation

Reply:
Btw, `one-var` is being dropped in the official MW style. We'll need to update our dependency soon.

Reply:
Oh nice! I'm actually not a huge fan of that rule so I'm not mad about it being dropped. Would you like me to create a bare task for that as a reminder?

Reply:
> Would you like me to create a bare task for that as a reminder?

por favor